include/dsn/tool-api/threads_info.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2017-present, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+
+#pragma once
+
+#include <dsn/tool-api/task.h>
+
+namespace dsn {
+namespace replication {
+
+class thread_info
+{
+public:
+    static std::vector<task_worker *> &get_info(int code);
+};
+
+} // namespace replication
+} // namespace dsn
```

Comment:
```suggestion
namespace dsn {

const std::vector<task_worker *> &get_task_code_threads_info(task_code code);

} // namespace dsn
```

There is nothing related to replication, so don't wrap the function in 'dsn::replication::' namespace.
Put this function to `dsn/tool-api/task.h`, since it's a utility of the task module.